.github/actions/ci/run-tests/action.yml

```diff
@@ -68,22 +68,28 @@ runs:
         $templateArgs = '${{ inputs.arguments }}'
         echo "TemplateArgs = '$templateArgs'"
         $dotnetVersion = $env:TargetDotNetVersion
-        $ValidationUnoCheckVersion = $env:ValidationUnoCheckVersionNet8
+        $ValidationUnoCheckVersion = $env:ValidationUnoCheckVersionNet10
```

Comment:
@jeromelaban Noticed the default UnoCheckVersion is Net10 while the default dotnetVersion is still Net9. Should this default instead align to Net9?